code/game/objects/items/toy_mechs.dm

```diff
@@ -446,19 +446,23 @@
 /obj/item/toy/prize/proc/check_battle_start(mob/living/carbon/user, obj/item/toy/prize/attacker, mob/living/carbon/target)
 	if(attacker?.in_combat)
 		to_chat(user, "<span class='notice'>[target?target.p_their() : "Your" ] [attacker.name] is in combat.</span>")
-		target?.to_chat(target, "<span class='notice'>Your [attacker.name] is in combat.</span>")
+		if(target)
+			to_chat(target, "<span class='notice'>Your [attacker.name] is in combat.</span>")
 		return FALSE
 	if(in_combat)
 		to_chat(user, "<span class='notice'>Your [name] is in combat.</span>")
-		target?.to_chat(target, "<span class='notice'>[target.p_their()] [name] is in combat.</span>")
+		if(target)
+			to_chat(target, "<span class='notice'>[target.p_their()] [name] is in combat.</span>")
```

Comment:
```suggestion
		to_chat(target, "<span class='notice'>[user.p_their()] [name] is in combat.</span>")
```
the fact that it uses your for the user but target's their for the target seems unintended?